NAMESPACE

```diff
@@ -52,3 +52,4 @@ importFrom(htmltools,tags)
 importFrom(htmlwidgets,JS)
 importFrom(htmlwidgets,saveWidget)
 importFrom(magrittr,"%>%")
+importFrom(promises,future_promise)
```

Comment:
I'd like the **promises** package to be a soft dependency (i.e., specified in `Suggests` rather than `Imports`).